src/components/pill/test/e2e/pill.spec.js

```diff
@@ -0,0 +1,86 @@
+import { assert, getDriver, config } from '../../../../utils/test';
+import { VlPill, VlButtonPill } from './pill.js';
+
+const { sbUrl } = config;
+const defaultUrl = `${sbUrl}?id=custom-elements-vl-pill--default`;
+const successUrl = `${sbUrl}?id=custom-elements-vl-pill--default&args=type:success`;
+const warningUrl = `${sbUrl}?id=custom-elements-vl-pill--default&args=type:warning`;
+const errorUrl = `${sbUrl}?id=custom-elements-vl-pill--default&args=type:error`;
+const closableUrl = `${sbUrl}?id=custom-elements-vl-pill--default&args=closable:true`;
+const checkableUrl = `${sbUrl}?id=custom-elements-vl-pill--default&args=checkable:true`;
+const pillButtonUrl = `${sbUrl}?id=custom-elements-vl-pill-vl-button-pill--default`;
+
+describe('vl-pill', async () => {
+  let driver;
+
+  beforeEach(() => {
+    driver = getDriver();
+    driver.manage().window().maximize();
+  });
+
+  it('als gebruiker kan ik de inhoud van een pill zien', async () => {
+    await driver.get(defaultUrl);
+    const pill = await new VlPill(driver, 'vl-pill');
```

Comment:
Strings die blijven terugkomen zoals `'vl-pill'` steek je best in een constante.

Reply:
Fixed.